Upgrade tempto version to 1.4

kokosing opened the pull request:
Upgrade tempto version to 1.4

kokosing committed:
Port tests to JAVA

kokosing committed:
Upgrade tempto version to 1.4

anusudarsan commented on presto-yarn-test/src/test/java/com/teradata/presto/yarn/test/slider/SliderStatusTest.java:
```diff
@@ -0,0 +1,45 @@
+        assertThat(sliderStatus.getLiveComponentsHost("COORDINATOR")).containsExactly("kogut-vsphere-default-master");
+
+        assertThat(sliderStatus.getLiveComponentsHost("WORKER")).containsExactly(
+                "kogut-vsphere-default-slave2",
```
this will fail in jenkins right?

kokosing replied:
no, note this is an unit test, see that travis passed

anusudarsan commented:
LGTM mod presto-td-yarn-product-tests-cdh passing

kokosing committed:
Run product tests on docker

CDH5 and HDP2.3 are used for testing.

Additionally:
 - do not run prerequiste fulfiller twice for the same cluster
 - fix checking memory heap size
 - do not check md5sum before uploading package
   It is no longer needed because:
    - package is different per each presto-yarn-test compilation
    - two copying files to docker is cheap

anusudarsan commented on presto-yarn-test/etc/docker/tempto-configuration-docker-local.yaml:
```diff
@@ -0,0 +1,15 @@
+ssh:
+  identity: /workspace/etc/docker/key
```
where is /workspace? Both here and in run_on_docker.sh?

anusudarsan commented on presto-yarn-test/src/main/java/com/teradata/presto/yarn/test/PrestoClusterTest.java:
```diff
@@ -221,14 +227,15 @@ public void multiNodeWithPlacementCheckingConnectors()
-            log.debug("Number of active nodes: " + String.valueOf(result.rows()));
-            return result.rows().equals(singletonList(prestoCluster.getAllNodes().size()));
+            QueryResult result = queryExecutor.executeQuery("select * from system.runtime.nodes");
+            Collection<String> allNodes = prestoCluster.getAllNodes()   ;
```
extra space before `;`
